labs/fabex.md

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+Fabex
+
+# Short Description
+The aim of this lab is to create a flexible block explorer for Hyperledger Fabric written in Go.
+
+# Scope of Lab
+Fabex aims to contribute to the Hyperledger Fabric audit ecosystem. It provides GRPC API and extensible database storage interface (MongoDB and CassandraDB implementations are now available). 
+This tool can be used as CLI or microservice. Simple API documented in [fabex.proto](https://github.com/VadimInshakov/fabex/blob/master/proto/fabex.proto). 
```

Comment:
Library exposes set of GRPC APIs to execute provenance queries over ledger data, moreover library provides an abstract DB interface allowing to integrate different DB engines, whereas out of box supported:

* MongoDB
* CassandraDB

The library could be integrated and deployed as a microservice, at the same time could be compiled into stand-alone command line tool.

Reply:
Ok, I fixed the Scope description.